pkg/agent/flowexporter/exporter/exporter_test.go

```diff
@@ -186,9 +187,12 @@ func testFlowExporter_sendDataSet(t *testing.T, v4Enabled bool, v6Enabled bool)
 			},
 		)
 		mockDataSet.EXPECT().GetSet().Return(dataSet)
-		mockIPFIXExpProc.EXPECT().AddSetAndSendMsg(ipfixentities.Data, dataSet).Return(0, nil)
-
-		err := flowExp.sendDataSet(mockDataSet, record)
+		mockIPFIXExpProc.EXPECT().SendSet(dataSet).Return(0, nil)
+		err := flowExp.addRecordToSet(mockDataSet, record)
+		if err != nil {
+			t.Errorf("Error in adding record to data set: %v", err)
+		}
```

Comment:
`assert.NoError(t, err, "Error when adding record to data set")`

Reply:
Done. Thanks!